contrib/gnuc2ooo.py

```diff
@@ -569,22 +569,22 @@ def MessageBox(MsgText, MsgTitle="GnuCash to Ooo", MsgType="messbox", MsgButtons
     global last_input_MessageBox
 ##    last_input_MessageBox = str(MsgText)
 
-    if not ParentWin:
-            doc = XSCRIPTCONTEXT.getDocument()
+    if not ParentWin:  # TODO: Where is XSCRIPTCONTEXT defined?
+            doc = XSCRIPTCONTEXT.getDocument()  # noqa: F821
```

Comment:
FYI, this script is intended to be used from within LibreOffice as a macro. LibreOffice will set the XSCRIPTCONTEXT variable before the macro is run.

Reply:
Then it is proper to put the statement __global XSCRIPTCONTEXT__ before this line of code.

Reply:
Considering this script doesn't make sense outside of LibreOffice, I'd rather make it so it exists gracefully if XSCRIPTCONTEXT is not defined. Something like this:

```
try:
    XSCRIPTCONTEXT
except NameError:
    print "This script can't be used standalone. Instead install it as a macro for LibreOffice and run it from there."
    exit
```

Reply:
__print()__ and __exit()__ are both functions in Python 3.

Reply:
Right. That was just from the top of my head to explain the idea.